package.json

```diff
@@ -36,10 +40,10 @@
     "supertest": "4.0.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-app-rewired eject"
```

Comment:
What is the advantage of `react-app-rewired`

Reply:
From antd documentation, it was listed to create a theme